src/firefly/js/ui/ColorPicker.jsx

```diff
@@ -63,25 +68,27 @@ function ColorPickerWrapper ({callback,color,callbackOnOKOnly, callbackOnBoth,
 
 
     return (
-        <div>
-            <div title={dialogTip}>
+        <Tooltip title={dialogTip} placement='right'>
```

Comment:
We need to set policy on Tooltip usage.  Is it always, or can `title` be used as well?